i18n/ind/docusaurus-plugin-content-docs/current/docs-user/staking/staking.md

```diff
@@ -1,136 +1,133 @@
 ```
 
-After the end of this 3-year span, we expect that developments in the Optimism Superchain Ecosystem (shared, decentralized sequencers) will lead us to a new staking mechanism.
+Setelah akhir periode 3 tahun ini, kami mengharapkan bahwa perkembangan dalam Ekosistem Optimism Superchain (sequencer terdesentralisasi bersama) akan membawa kami ke mekanisme staking baru.
 
 ## Unstaking
 
-Once the locking period of a [locking position](#locking-positions) has expired, it is possible to unlock the staked tokens again, by [unstaking](stake-unstake.mdx#how-to-unstake-your-lsk-tokens) them.
+Setelah periode kunci dari sebuah [posisi kunci](#locking-positions) berakhir, Anda dapat membuka kembali token yang di-stake dengan cara [unstake](stake-unstake.mdx#how-to-unstake-your-lsk-tokens).
 
 :::tip
-In case you need to unlock your tokens earlier than the defined locking period, it is possible to unstake your tokens before the locking period expires by using the [early unlock](#early-unlock) option.
+Jika Anda perlu membuka token Anda lebih awal dari periode kunci yang ditentukan, Anda dapat melakukan unstake token Anda sebelum periode kunci berakhir dengan menggunakan opsi [early unlock](#early-unlock).
 
-Using fast unlock will involve a penalty to be paid.
+Menggunakan unlock cepat akan melibatkan penalti yang harus dibayar.
 :::
 
-Once they are unstaked, the LSK tokens can be used as before for any other purpose.
+Setelah di-unstake, token LSK dapat digunakan seperti sebelumnya untuk tujuan lainnya.
 
-### Early Unlock
-Early unlock allows you to unstake your LSK tokens before their locking period has expired.
-It involves paying a penalty and waiting for a 3-day emergency unlocking period.
-The penalty is proportional to the remaining days in the staking period that will be skipped.
-
-The penalty is calculated using the formula:
+### Unlock Cepat
+Unlock cepat memungkinkan Anda untuk melakukan unstake token LSK sebelum periode kunci mereka berakhir.
+Ini melibatkan pembayaran penalti dan menunggu periode unlock darurat selama 3 hari.
+Penalti dihitung berdasarkan formula:
 
 ```
-Penalty = 0.5 * Staked Amount * (Remaining Days / 730)
+Penalti = 0.5 * Jumlah yang di-stake * (Hari Tersisa / 730)
 ```
 
-Please note that the maximum value for the remaining days is 730.
+Harap dicatat bahwa nilai maksimum untuk hari yang tersisa adalah 730.
 
-For example, if a user wants to fast unlock a position with 100 LSK and a remaining locking duration of 150 days, then the penalty is calculated as follows:
+Misalnya, jika seorang pengguna ingin melakukan unlock cepat posisi dengan 100 LSK dan durasi kunci yang tersisa 150 hari, maka penalti dihitung sebagai berikut:
 
 ```
-Penalty = 0.5 x 100 x (147 / 730) = 10.068 LSK
+Penalti = 0.5 x 100 x (147 / 730) = 10.068 LSK
 ```
 
-Note that we multiply by 147 and not 150, since the user will still have to wait for 3 days, so the locking duration decreased by 147 overall. 
-
```

Comment:
@TalhaMaliktz "so the locking duration decreased by" -> "so the locking duration is decreased by"

Reply:
@TalhaMaliktz are you sure it is "decreased by 147 overall", not "decreased by 3 overall"? 🤔

Reply:
@fujianggara10 may need to change the Indonesian translation, based on Talha's reply later.